go.mod

```diff
@@ -212,3 +225,5 @@ exclude (
 	github.com/grpc-ecosystem/grpc-gateway v1.14.7
 	google.golang.org/api v0.30.0
 )
+
+replace github.com/open-telemetry/opentelemetry-collector-contrib/processor/deltatocumulativeprocessor => github.com/sh0rez/opentelemetry-collector-contrib/processor/deltatocumulativeprocessor v0.0.0-20241011125442-ac7080f67eb9
```

Comment:
This is replaced until https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/35562 is merged, which fixes a memory-reuse bug that causes deltatocumulative to modify the first sample it emitted later on.